treefmt.nix

```diff
@@ -0,0 +1,32 @@
+{ lib, pkgs, ... }:
+
+{
+  tree-root-file = ".git/config";
+  on-unmatched = "fatal";
+
+  excludes = [
+    "*.lock"
+    "*.md"
+    ".gitignore"
+    "LICENSE"
+  ];
+
+  formatter.nixfmt = {
+    command = lib.getExe pkgs.nixfmt;
+    includes = [ "*.nix" ];
+    options = [ "--strict" ];
+  };
+
+  formatter.prettier = {
```

Comment:
FYI: `biome` seems to be getting more popular in treefmt configs. Apparently it is very similar to prettier, but faster.